### DISCUSSION TITLE
Doom Integration? Biblio module?

### DISCUSSION THREAD

**Discussion by lguenth** [emacs-citar/citar#86] [Q&A] [answered]
Hey, thanks for providing this package! I haven't tried it yet, but as a grad student, this looks extremely promising!
I know you asked for better referencing support in Doom and I already use the biblio module... Are there any plans to integrate bibtex-actions into that?

Best regards
Luke

**Comment by bdarcus** [comment, marked as answer]
Hi @lguenth - absolutely!

It's ready to go, as soon as the new selectrum module is merged, and @hlissner opens up PRs again ;-)

https://github.com/bdarcus/.doom.d/tree/main/modules/tools/biblio

I understand that may be awhile, but I'll try to keep my local module updated for people that want this (I already see a change I need to make).

**Comment by bdarcus** [comment]
@lguenth - as of today, this package is integrated in the WIP selectrum module PR. As in, when merged, the `biblio` module will include the update.

https://github.com/hlissner/doom-emacs/pull/4664
